RELEASE/scripts/autoscend.ash

```diff
@@ -2824,14 +2825,9 @@ void auto_begin()
 			auto_log_warning("Okay, but the warranty is off.", "red");
 		}
 	}
-
+	
 	//This also should set our path too.
 	string page = visit_url("main.php");
```

Comment:
```suggestion
	string page = visit_url("main.php");
	page = visit_url("api.php?what=status&for=4", false);
```
you need to restore both deleted lines not just visiting main.
without visiting the char page then the string page will not contain the info it needs for the subsequent lines to fire off.
this will break picky, jello, bat, and astral spirit.

Reply:
actually visiting main might be superfluous or might be a necessity for initializing mafia properly after ascension.
however visiting the character page is definitely required.